Common/include/config_structure.hpp

```diff
@@ -723,6 +733,7 @@ class CConfig {
   Wrt_Limiters,              /*!< \brief Write residuals to solution file */
   Wrt_SharpEdges,              /*!< \brief Write residuals to solution file */
   Wrt_Halo,                   /*!< \brief Write rind layers in solution files */
+  Wrt_Slice,                   /*!< \brief Write 1D slice of a 2D cartesian solution */
```

Comment:
What is the reason for this?

Reply:
It's meant to be a hidden feature :). If you have a cartesian 2D grid (structured), I have put in a routine that will extract profiles from lines of nodes with a constant X or Y coordinate. This is particularly helpful for V&V cases, like the NASA turbulence modeling cases for flat plates, etc., where you want to extract the various profiles for comparison with other codes and experiment.